tests/comfy_cli/jobs/test_jobs.py

```
@@ -2559,6 +2637,257 @@ def test_watch_executing_escapes_server_controlled_node_markup():
+    def recv(self):
+        if not self._messages:
+            raise jobs_mod.WebSocketTimeoutException("timed out")
+        return self._messages.pop(0)
+
+    def close(self):
+        self.closed = True
+
+
+def _run_local_watch(monkeypatch, capsys, *, messages, prompt_id="pid-w", argv_extra=()):
+    """Drive `jobs watch` (local, NDJSON) over a scripted WS; return the lines."""
+    from typer.testing import CliRunner
```

Pylint (4.0.6)

[convention] Import outside toplevel (typer.testing.CliRunner)

(C0415)